docs/competition/team-support/helpdesk.md

```diff
@@ -0,0 +1,110 @@
+ * Spare robot badge for teams to borrow to try for size (must be clearly marked and returned to helpdesk ASAP)
+ * Saturday evening battery/charger loans
+ * Velcro
+ * Scissors
+ * Velcro measurement/instruction sheet
+
+## Volunteer Requirements
+
+At least one volunteer on Helpdesk must have prior experience of the use of the Student Robotics kit.
+
+## Procedures
+
+Almost all of Helpdesk's operations are covered by the following procedures.
+
+### Handling of all requests
+
+Whenever a team comes to Helpdesk with an issue a Helpdesk Volunteer must fill in their TLA and a single-sentence description of the issue on an Issue Form. When the issue has been resolved a single sentence description of the resolution must be written on the form. This is primarily for analytical purposes so that we can improve the operation of Helpdesk in future years.
+
+Whenever a team comes to Helpdesk with an issue that cannot be solved instantly their TLA and a very brief (couple of words) summary of the problem must be written on a post-it note and stuck up on a board of current issues. This ensures that no problems get dropped while they are being investigated. When the issue is resolved the post-it note can be thrown away. If it appears that a particular post-it has remained on the board for an extended period of time and there has been no obvious activity to resolve the issue then a volunteer should be dispatched to the team to see if it's still a problem.
+
+### Handling of requests for robot safety checks
+
+The robots require safety checking before they are allowed to compete in the competition. Teams will not be given their hi-viz vest until their robot has passed the safety check procedure. Helpdesk volunteers should direct teams enquiring about safety checks to the Health & Safety Coordinator (Scarzy).
+
+### Handling of general robot development requests
+
+Many teams will come to Helpdesk requesting help with the development of their robot or, more likely, help using the Student Robotics kit and associated software. As mentioned earlier, Helpdesk volunteers are not expected to tell teams what to do, but rather offer advice. Helpdesk volunteers should strive to answer questions about the use of the kit and software.
+
+Some queries may require a volunteer to visit the team in their pit to observe behaviour in situ. This must only be done by a Helpdesk volunteer if it will leave no fewer than two volunteers actively manning Helpdesk. If it is not possible for a Helpdesk volunteer to visit a team then a [Roving Helper](/competition/team-support/Roving_Helper) should be summoned, via radio if necessary.
+
+In some rare situations it may be necessary to request the assistance of a person with more in-depth knowledge of the kit to help solve an issue. This should only be done if the issue is non-trivial. The following people are available for specific help and can be summoned over the radio:
+
+Electronic Hardware
```

Comment:
Whilst an escalation tree is a useful thing, this definitely isn't one. 

https://github.com/srobo/tasks/issues/150 covers creating this. I suggest we remove it from here for now, and add it back once we have one.

Reply:
I suspect it will be a per-year thing as it depends on who is going to be around.